homework3/semi-uber/Server/api/Schemas/load.schema.js

```diff
@@ -1,28 +1,26 @@
 const mongoose = require("mongoose");
 const Joi = require("@hapi/joi");
 const Schema = mongoose.Schema;
+const { STATUS } = require("../../data/loadData.json");
+const dimensionsSchema = require("./dimensions.schema");
 
 const Load = new Schema({
   created_by: { type: String },
   logs: { type: Array },
-  assigned_to: { type: String },
-  status: { type: String },
-  state: { type: String },
+  assigned_to: { type: String, default: "Not assigned" },
```

Comment:
I guess that default should be `null`. This way it will be easier for you to check if `Load` is assigned to someone

Reply:
done